CMakeLists.txt

```diff
@@ -13,8 +13,12 @@
 cmake_minimum_required(VERSION 2.8.4 FATAL_ERROR)
 cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION})
 
+IF (NOT DEFINED KFS_DIR_PREFIX)
+    set (KFS_DIR_PREFIX ${CMAKE_SOURCE_DIR})
```

Comment:
This isn't really external project, but rather "internal" project / code that it not in the QFS tree (it was in the QFS tree prior to Quantcast open sourcing). As such the code relies on the QFS source code, instead of presently "exported" / "external" / "public" [client] interface.